Lib/glyphsLib/__main__.py

```diff
@@ -14,48 +14,45 @@
 
 from __future__ import print_function, division, absolute_import, unicode_literals
 
-import sys
 import argparse
+import os
+import sys
 
-import glyphsLib
+import glyphsLib.cli
 
 
-description = """\n
-Converts a Glyphs.app source file into UFO masters
-or UFO instances and MutatorMath designspace.
-"""
+def main():
+    if len(sys.argv) >= 2:
+        # Relay to subcommand.
```

Comment:
clever, doesn't argparse already supports subcommands?

Reply:
Mh, yes. So I'm going to do the argparsing in main instead of the `*2*` functions.

Reply:
The separate ufo2glyphs and glyphs2ufo scripts need to go then I guess, as there is only main() and no parsing is done in the functions as such?